src/renderPage.js

```diff
@@ -0,0 +1,90 @@
+import { images } from './app.js';
+import { MENU } from './app.js';
+import { movieUrl } from './app.js';
+import { settingsGet } from './app.js';
+
+async function renderPage(response, page) {
```

Comment:
This function contains different logic for three different pages. This means that it a joint between different unrelated pieces of code, tying them together, weakening the separation of concerns.

I would have broken this into three different functions, or moved the parts of the function which are different for the three cases to the routes that invoke the function instead.